src/OpenTelemetry.Exporter.OpenTelemetryProtocol.Grpc/OpenTelemetry.Exporter.OpenTelemetryProtocol.Grpc.csproj

```
@@ -0,0 +1,17 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <!-- OmniSharp/VS Code requires TargetFrameworks to be in descending order for IntelliSense and analysis. -->
+    <TargetFrameworks>netstandard2.0</TargetFrameworks>
+    <Description>OpenTelemetry protocol exporter over gRPC for OpenTelemetry .NET</Description>
+    <PackageTags>$(PackageTags);OTLP;gRPC</PackageTags>
+    <MinVerTagPrefix>core-</MinVerTagPrefix>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="$(RepoRoot)\src\OpenTelemetry\OpenTelemetry.csproj" />
+    <PackageReference Include="Google.Protobuf" Version="$(GoogleProtobufPkgVer)" />
+    <PackageReference Include="Grpc.Tools" Version="$(GrpcToolsPkgVer)" PrivateAssets="all" />
+    <PackageReference Include="Grpc.Net.Client" Version="$(GrpcNetClientPkgVer)" />
+  </ItemGroup>
```

nit: we usually put PackageReferences and ProjectReferences in their own ItemGroups... though, admittedly we have probably not kept completely consistent.

```suggestion
  <ItemGroup>
    <PackageReference Include="Google.Protobuf" Version="$(GoogleProtobufPkgVer)" />
    <PackageReference Include="Grpc.Tools" Version="$(GrpcToolsPkgVer)" PrivateAssets="all" />
    <PackageReference Include="Grpc.Net.Client" Version="$(GrpcNetClientPkgVer)" />
  </ItemGroup>
  
  <ItemGroup>
    <ProjectReference Include="$(RepoRoot)\src\OpenTelemetry\OpenTelemetry.csproj" />
  </ItemGroup>
```